src/views/statusView.ts

```diff
@@ -124,6 +132,7 @@ export default class StatusView implements vscode.Disposable {
         this.showStatusBarItem(fileUri, bar.statusQuery);
         this.showStatusBarItem(fileUri, bar.statusLanguageService);
         this.showStatusBarItem(fileUri, bar.sqlCmdMode);
+        this.showStatusBarItem(fileUri, bar.rowCount);
```

Comment:
Is this going to look fine if the row count item doesn't have any text? i.e. if an error occurs or you switch away from a tab before executing a query?